### DISCUSSION TITLE
Any opinion on writing weekly note on Trilium?

### DISCUSSION THREAD

**Discussion by shutaozhenzhen** [TriliumNext/Trilium#3534] [General]
Trilium calender do not have note for weekly note. As one week is a period of work, I still want to create week note.
But it seems hard to combine it with Trilium, as a week can in both month.
Any opinion on this?

**Comment by zadam** [comment]
Hi, if you really want it and don't mind a bit of manual reorganization, then it can be done. The rigid structure of Year -> Month -> Day is used only when creating day notes. Once they are created, they can be moved around as you wish - you can create the week notes yourself and move the day notes there manually. All functionality like "Today" button will keep working since they care only about day notes.

**Comment by devlinjunker** [comment]
You would need to modify the default scripts a bit....

but I think you could create a separate `Weeks` folder in the Year of the Journal that clones the notes from the Month section to the specific week

Something like this:
<img width="376" alt="Screen Shot 2023-01-19 at 2 32 04 PM" src="https://user-images.githubusercontent.com/1504590/213577229-beeedcc6-e182-4d58-861d-cbc79e9a93f0.png">

**Comment by zadam** [comment]
Somebody is working on a PR to add week support here: https://github.com/zadam/trilium/pull/3544
